### DISCUSSION TITLE
把自定义主界面功能图形化

### DISCUSSION THREAD

**Discussion by Skying-zhishi** [Meloong-Git/PCL#851] [提问] [answered]
把自定义主界面功能图形化

**Comment by WForst-Breeze** [comment, marked as answer]
如有新功能建议，请提交 新功能提案 ，不过你提及这个功能的感觉制作的可能性不大（

补充：已经有第三方软件制作了此功能，PCL2 不考虑制作此功能

**Reply by WForst-Breeze** [reply]
@LTCatt 再次打扰（）麻烦看看这个功能是否有计划制作，我记得好像以前有人提及过，但具体是在哪找不到了，麻烦猫猫了qwq

**Reply by LTCatt** [reply]
我不可能做……
我记得已经有第三方做过了？

**Reply by xuzehao2012** [reply]
我觉得pcl 应该这样

**Reply by xuzehao2012** [reply]
应为第三方支持不好，不能自定义，更新慢，bug多，况且哪有那么好找

**Reply by xuzehao2012** [reply]
萌新不知道就只能羡慕

**Reply by xuzehao2012** [reply]
网上不是万能的，不行信去搜

**Reply by WorldHim** [reply]
> 我觉得pcl 应该这样

PCL 做新功能不是只要某人主观臆断就行了。目前来说确实没有必要做这个功能，而且龙猫已经表过态，所以就别再吵了，谢谢。

**Reply by WForst-Breeze** [reply]
> 我觉得pcl 应该这样

~~错别字好多看得我好费劲qwq~~ 再次强调，PCL2 是 MC 启动器，开发者只会选择制作相对而言优先级较高的功能，并不是说这项功能不好才不去做的，况且自定义主页定位就算是**比较进阶的功能**，自然无法完全照顾到萌新们，但我相信只要耐心看看教学肯定高低能做出来一些不错的样式的。
~~虽然但是，这种生成器网上真是一抓一大把……搜不到真的没办法~~

**Comment by miner233** [comment]
WPF 控件而已，你可以自己到网上搜 WPF 控件的详解或是图形化工具，PCL2 做它意义不大...

**Comment by xuzehao2012** [comment]
你就别做呗

**Comment by xuzehao2012** [comment]
启动器就是启动器，搞得花里胡哨的
